UndertaleModTool/Scripts/Builtin Scripts/RunSwitchAndXboxOnPC.csx

```diff
@@ -0,0 +1,172 @@
+// Sets up NXTALE to run well on PC.
+using System.Linq;
+using UndertaleModLib.Models;
+
+EnsureDataLoaded();
+
+if (Data.GeneralInfo.Name.Content != "NXTALE")
+{
+    ScriptError("""
+        This script can only be used with the Nintendo Switch
+        and Xbox One version of Undertale.
+        """, "Not NXTALE");
+    return;
+}
+
+bool isXbox = Data.Rooms.ByName("room_xbox_engagement") is not null;
+
+if (isXbox)
+{
+    // Fix initialization code not running
+    ReplaceTextInGML("gml_Object_obj_time_Create_0", """
+    if (os_type == os_xboxone)
+    {
+        script_execute(SCR_GAMESTART, 0, 0, 0, 0, 0)
+    """, """
+    if (true)
+    {
+        script_execute(SCR_GAMESTART, 0, 0, 0, 0, 0)
+    """);
+
+    var obj_time = Data.GameObjects.ByName("obj_time");
+    var otherEvents = obj_time.Events[(int)EventType.Other];
+    var gameStartEvent = otherEvents.FirstOrDefault(x => x.EventSubtype == (int)EventSubtypeOther.GameStart);
+    if (gameStartEvent is not null)
+        otherEvents.Remove(gameStartEvent);
```

Comment:
If this is always expected to be there (which it should be), just make it `otherEvents.Remove(otherEvents.First(x => x.EventSubtype == int(EventSubtypeOther.GameStart))`

Alternatively, you can also do something akin to `obj_time.EventHandlerFor(EventType.Other, EventSubtypeOther.GameStart, Data).ReplaceGML("");`

Reply:
I like to run the script several times in a debugging session